engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/unboxing/SuspendedFieldGetterNode.java

```diff
@@ -0,0 +1,66 @@
+package org.enso.interpreter.runtime.callable.atom.unboxing;
+
+import com.oracle.truffle.api.exception.AbstractTruffleException;
+import com.oracle.truffle.api.interop.TruffleObject;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.node.callable.InvokeCallableNode;
+import org.enso.interpreter.node.callable.dispatch.InvokeFunctionNode;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.callable.argument.CallArgumentInfo;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+import org.enso.interpreter.runtime.callable.function.Function;
+import org.enso.interpreter.runtime.state.State;
+
+/**
+ * Getter node that reads a field value. If the value is a thunk the node
+ * evaluates it and replaces the original lazy value with the new value.
+ *
```

Comment:
Seems like an unnecessary newline here?

```suggestion
```

Reply:
Done in [3619f8a](https://github.com/enso-org/enso/pull/6151/commits/3619f8a88b0063239dddce5896b724d471897043)